paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/shared/MultiToggleButton.kt

```diff
@@ -46,10 +50,20 @@ fun MultiToggleButton(
                 )
             }
 
-            Row(
+            Text(
+                toggleState,
+                color = textColor,
                 modifier = Modifier
-                    .background(backgroundTint)
-                    .padding(vertical = 6.dp, horizontal = 8.dp)
+                    .background(
+                        color = backgroundTint,
+                        shape = RoundedCornerShape(
+                            topStart = if (index == 0) 4.dp else 0.dp,
+                            bottomStart = if (index == 0) 4.dp else 0.dp,
+                            topEnd = if (index == toggleStates.size - 1) 4.dp else 0.dp,
+                            bottomEnd = if (index == toggleStates.size - 1) 4.dp else 0.dp,
+                        )
+                    )
+                    .padding(vertical = 4.dp, horizontal = 8.dp)
                     .toggleable(
```

Comment:
We had lots of places in PaymentSheet where we set padding before setting a click listener, which results in an odd-looking ripple effect. Let’s set the click listener (here: `toggleable`) first.